.github/workflows/release.yml

```diff
@@ -250,6 +250,7 @@ jobs:
         run: .github/workflows/scripts/unix_test.sh
         env:
           TI_WANTED_ARCHS: 'cpu'
+          TI_SKIP_CPP_TESTS: Disabled because Vulkan is supported but not working on buildbot4
```

Comment:
hmmm I thought this was guarded by https://github.com/taichi-dev/taichi/blob/master/tests/cpp/aot/vulkan/aot_save_load_test.cpp#L220? Is there any particular reason that we want to disable it explicitly?

Reply:
1. On buildbot4 vulkan IS available but if you use it wherever it will crash.
2. Nightly builds are currently broken because of this.

Reply:
This is a temporary workaround and I'll try fix it by https://github.com/taichi-dev/taichi/issues/5544